cmd/ebpf/xdp/utils/gtp_utils.h

```diff
@@ -28,7 +28,7 @@
 
 static __always_inline __u32 parse_gtp(struct packet_context *ctx) {
     struct gtpuhdr *gtp = (struct gtpuhdr *)ctx->data;
-    if ((void *)(gtp + 1) > ctx->data_end)
+    if ((char *)(gtp + 1) > ctx->data_end)
```

Comment:
`data_end` is `const char*`, so seems like `(const char *)` convertion should be used instead `(char *)`